internal/util/walg/backups.go

```diff
@@ -201,6 +201,17 @@ func DeleteBackup(
 	return result, err
 }
 
+func DeleteAllBackupsAndWALsInStorage(
+	ctx context.Context,
+	backupConfig *v1beta1.BackupConfigWithSecrets,
+	pgMajorVersion int,
+) (*cmd.RunResult, error) {
+	return cmd.New("wal-g", "delete", "everything", "--confirm").
```

Comment:
delete everything won't delete permanent backups unless `FORCE` modifier specified

Reply:
Sure, thanks, I'll fix this

Reply:
Fixed in 3bb8009cc96e1312fd21650a1ac0c044d394c73e